packages/roosterjs-content-model-plugins/lib/autoFormat/list/keyboardListTrigger.ts

```diff
@@ -13,26 +13,28 @@ export function keyboardListTrigger(
     shouldSearchForBullet: boolean = true,
     shouldSearchForNumbering: boolean = true
 ) {
-    editor.formatContentModel((model, _context) => {
-        const listStyleType = getListTypeStyle(
-            model,
-            shouldSearchForBullet,
-            shouldSearchForNumbering
-        );
-        if (listStyleType) {
-            const segmentsAndParagraphs = getSelectedSegmentsAndParagraphs(model, false);
-            if (segmentsAndParagraphs[0] && segmentsAndParagraphs[0][1]) {
-                segmentsAndParagraphs[0][1].segments.splice(0, 1);
-            }
-            const { listType, styleType, index } = listStyleType;
-            triggerList(editor, model, listType, styleType, index);
-            rawEvent.preventDefault();
-            normalizeContentModel(model);
+    if (shouldSearchForBullet || shouldSearchForNumbering) {
```

Comment:
same here, check outside to reduce unnecessary parameters.

Reply:
We already use these parameter on line 20 and 21, then we cannot remove them

Reply:
Should I return this check to outside the function anyway?

Reply:
I see. In that case we can keep it